src/public/curriculum/growing-growing-growing/growing-growing-growing.json

```diff
@@ -180,6 +180,34 @@
           "title": "GGG 1.3 Killer Plant Strikes Lake Victoria",
           "subtitle": "y-intercepts Other than 1",
           "sections": [
+                        {
+                        "type": "introduction",
+                        "content": {
+                            "tiles": []
+                        },
+                        "supports": []
```

Comment:
Indentation

Reply:
Done, I've corrected the indentation. Thanks for catching that.

Reply:
FYI, we generally use two-space tab indents, but it's not worth going back to fix it at this point.